ui/src/components/PostPreview.tsx

```diff
@@ -0,0 +1,70 @@
+import * as React from "react";
+import cx from "classnames";
+import { Post } from "../queries/posts";
+import { Button } from "./Button";
+import { Skill, skillInfoMap } from "../model/skill";
+import { SkillIcon } from "./SkillIcon";
+
+interface Props {
+  post: Post;
+  className?: string;
+}
+
+export const PostPreview: React.FC<Props> = ({ post, className }) => {
+  return (
+    <article
+      className={cx(
+        "border-2 border-white p-2 grid grid-flow-row auto-cols-auto gap-y-2",
+        className
+      )}
+    >
+      <h3 className="font-bold text-xl">[title goes here]</h3>
```

Comment:
The API now actually returns the post title, so it could be added in this PR if you'd like